silverback/_cli.py

```diff
@@ -433,6 +519,7 @@ def new_bot(
     network: str,
     account: str | None,
     vargroups: list[str],
+    docker_credentials_name: str,
```

Comment:
This is none by default?
```suggestion
    docker_credentials_name: str | None,
```